sal_interfaces/MTHexapod/MTHexapod_Commands.xml

```diff
@@ -24,7 +24,7 @@
     <Description>Set position and orientation limits.</Description>
     <item>
       <EFDB_Name>maxXY</EFDB_Name>
-      <Description>Maximum |X| and |Y|</Description>
+      <Description>Maximum X and Y</Description>
```

Comment:
These bars indicate the absolute value, so I would prefer to keep them if possible. Why are they triggering warnings in the documentation generation step? Can you fix that step, instead?